src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -898,11 +898,11 @@ public function iRequestTheRemovalOfCatalogPromotion(CatalogPromotionInterface $
     }
 
     /**
-     * @When I request the archival of :catalogPromotion catalog promotion
+     * @When I request the archivation of the :catalogPromotion catalog promotion
      */
-    public function iRequestTheArchivalOfCatalogPromotion(CatalogPromotionInterface $catalogPromotion): void
+    public function iRequestTheArchivationOfCatalogPromotion(CatalogPromotionInterface $catalogPromotion): void
```

Comment:
Either add `the` over here or remove `the` in `src/Sylius/Behat/Context/Ui/Admin/ManagingCatalogPromotionsContext.php` `iRequestTheArchivationOfTheCatalogPromotion`

```suggestion
    public function iRequestTheArchivationOfTheCatalogPromotion(CatalogPromotionInterface $catalogPromotion): void
```